bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -505,13 +505,30 @@ impl Room {
     pub async fn typing_notice(&self, is_typing: bool) -> Result<(), ClientError> {
         Ok(self.inner.typing_notice(is_typing).await?)
     }
+    pub async fn subscribe_to_typing_notifications(
```

Comment:
Please; make the code to breath:

```suggestion
    }

    pub async fn subscribe_to_typing_notifications(
```